source/special.tex

```diff
@@ -3372,46 +3372,46 @@
 of the defaulted three-way comparison operator function
 with parameters \tcode{x} and \tcode{y} of the same type
 is determined by comparing corresponding elements
-\tcode{x}$_i$ and \tcode{y}$_i$
+$\tcode{x}_i$ and $\tcode{y}_i$
 in the expanded lists of subobjects for \tcode{x} and \tcode{y}
 until the first index $i$
-where \tcode{x}$_i$ \tcode{<=>} \tcode{y}$_i$
-yields a result value \tcode{v}$_i$ where \tcode{v}$_i$\tcode{ != 0},
+where $\tcode{x}_i$ \tcode{<=>} $\tcode{y}_i$
+yields a result value $\tcode{v}_i$ where $\tcode{v}_i \tcode{!= } 0$,
```

Comment:
I take it you've checked that the spacing after != looks right?

Reply:
eek. Add a `\,`-space before?

Reply:
Or see how it looks without the space in the tcode.

Reply:
I propose using `\mathrel`: `$\tcode{v}_i$ where $\tcode{v}_i \mathrel{\tcode{!=}} 0$`.  `\mathrel` provides the same spacing around its contents as TeX provides around other math relations (e.g., =, >).

Reply:
Just plain spaces works:
![space1](https://user-images.githubusercontent.com/23412755/38781059-3ac85aa6-40e0-11e8-802e-6f796d6da6f5.png)

Reply:
And, even better, with \mathrel:
![space1](https://user-images.githubusercontent.com/23412755/38781108-e041cec2-40e0-11e8-843c-fdf9849d2b43.png)

Update pushed.

Reply:
Agreed, \mathrel is the right thing. Thanks, Kevin!